### DISCUSSION TITLE
Migrating from Azure DevOps Repos to Github private repos

### DISCUSSION THREAD

**Discussion by westerdaled** [community/community#22317] [New to GitHub]
<p>Hi</p>
<p>I have a number of small personal  repos that are currrently managed under DevOps . Alhough I could leave them where they are, rehousing them as private repos on GitHub seems a better location govem they are for personal use. It has a similar interface when considering personal use. Also public and private repos can be administered in a consistent way.</p>
<p>Are there migration steps documented anywhere, covering git commands  to prep the local repo and the review so show all commits are moved accross?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Yes, there is! See <a href="https://help.github.com/articles/importing-a-git-repository-using-the-command-line/" rel="nofollow noopener">https://help.github.com/articles/importing-a-git-repository-using-the-command-line/</a> . Please let us know if you need any more help.</p>

**Comment by uraut-spurqlabs** [comment]
Using azure pipeline you can migrate your repositories into GitHub. 
https://spurqlabs.com/how-to-set-an-automatic-backup-of-your-code-repository/

**Comment by schatur** [comment]
Hi Team,

After migrating azure repos to github if i don't want see history for perticular file  in github what is the solution ? please suggest

**Reply by mpboom** [reply]
Hey @schatur it probably works best if you open up a new topic, but in any case you want to remove the history from a certain file if I understand correctly? Unless you're willing to rewrite your entire repositories' commit history up until that point, I'm afraid that can't be done.

**Reply by igor9silva** [reply]
@schatur if issue is an exposed value, I'd recommend archiving that repo and starting a new one, with a new history.

I know I am late to the discussion so, did you any managed to achieve your original intention?
